.babelrc

```diff
@@ -0,0 +1,20 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4"
```

Comment:
I think we could increment this to 6, since we don't support unmaintained Node.js versions.